generated/thrift/rpc.thrift

```diff
@@ -83,6 +84,8 @@ struct FetchBlocksMetadataRequest {
 
 struct FetchBlocksMetadataResult {
 	1: required list<BlocksMetadata> elements
+	2: optional i64 nextToken
```

Comment:
Can we call this `nextPageToken` to match the request `pageToken` naming scheme?